.golangci.yml

```diff
@@ -49,8 +49,7 @@ linters-settings:
         # relative to the directory that golangci-lint is invoked in.
         # rules.go
         # - "rules.go" -> Load rules.go for each linted go.mod file due to taskfile cd'ing into each module.
-        # - "${configDir}/pkg/lint/rules/*.go" -> Load all go files in pkg/lint/rules.
-        rules: "rules.go,${configDir}/pkg/lint/rules/*.go"
```

Comment:
"Global" rules should not be discovered by golangci? With this change rules need to be copied per go module?

Reply:
This was removed due to `src/go/k8s` being picked up by the gob. When evaluated at that path pkg/lint/rules doesn't get loaded correctly and errors get thrown if `*.go` matches no files. I may just rip this linter out as it only worked in 50% of the cases I actually needed it in because it doesn't handle generics :(